docs/source/tutorial.rst

```diff
@@ -81,7 +81,7 @@ An input dataframe should meet the following constraints:
   - Minimum required samples depends on number of parameters in the chosen models. run_forecast() will fail to fit if
     n_samples < n_parameters+2
   - y column may include null values.
-  - Multiple values per sample may be included, e.g when the same metric is observed by multiple sources. In that case,
+  - Multiple values per sample may be included, e.g. when the same metric is observed by multiple sources. In that case,
     it is possible to assign weights to each individual sample so that one source will be given higher priority. To do
```

Comment:
Does it get to the next line here?

Reply:
if it's a continuing line no problem then. It seemed that there were more spaces there so the line was breaking without an actual reason...